content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield-nodestore.js

```diff
@@ -248,12 +252,20 @@
     $document.ready(function () {
         var compositeMultiField = new ACS.TouchUI.NodeCompositeMultiField();
 
-        $document.on("dialog-ready", function(){
+        if (onPropertiesPage()) {
```

Comment:
i think we can move this common function to touchui-widgets-init.js and call it from both multifield files (json and node store) 

`compositeMultiField.isOnPropertiesPage()`